library/Icingadb/Widget/Detail/CheckStatistics.php

```diff
@@ -169,6 +169,15 @@ protected function assembleFooter(BaseHtmlElement $footer)
             t('Scheduling Source') . ':',
             $this->object->state->scheduling_source ?? (new EmptyState(t('n. a.')))->setTag('span')
         ));
+
+        if ($this->object->timeperiod->id) {
+            $footer->add(new HorizontalKeyValue(
+                t('Timeperiod') . ':',
+                $this->object->timeperiod->display_name ?? $this->object->timeperiod->name
+            ));
+
+            $footer->addAttributes(['class' => 'timeperiod']);
```

Comment:
Please use a different name for the class. The footer itself is not a *timeperiod*, it rather *has-a-timeperiod*, but that's non-sense as well. Why not call the class after what it does?